### PR TITLE
GameCube Save Management

### PR DESCRIPTION
**Pull request by playerjmr**
added guide for managing gamecube saves on the wii.

also added a typo fix in the update page

### CODE REVIEW

File: _pages/en_US/site-navigation.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ sitemap: false
 - [Installing RSSMii](rssmii)
 - [Installing Wii Menu Themes](themes)
 - [Installing Wii Menu Themes on vWii](themes-vwii)
+- [Managing Gamecube Save Files] (gcsaves)
```
Comment (osfanbuff63) [inline _pages/en_US/site-navigation.md:60]:
```suggestion
- [Managing Gamecube Save Files](gcsaves)
```
Markdown syntax requires this to not have a space

**Comment by larsenv** [comment]
Good job. Can you please proofread your tutorial? Nintendo's spelled as nintnedont in one place and GameCube isn't capitalized in one other place.

**Comment by playerjmr** [comment]
> Good job. Can you please proofread your tutorial? Nintendo's spelled as nintnedont in one place and GameCube isn't capitalized in one other place.

nintendont was intended. however I will re-write it as Nintendon't. ive also fixed various grammar issues. how do i add these changes to the current pull request?
[gcsaves.md](https://github.com/RiiConnect24/Wii-Guide/files/11291906/gcsaves.md)

**Comment by Snare-Hawk** [comment]
> > Good job. Can you please proofread your tutorial? Nintendo's spelled as nintnedont in one place and GameCube isn't capitalized in one other place.
> 
> nintendont was intended. however I will re-write it as Nintendon't. ive also fixed various grammar issues. how do i add these changes to the current pull request? [gcsaves.md](https://github.com/RiiConnect24/Wii-Guide/files/11291906/gcsaves.md)

all you have to do is push the changes like you did before
also, any instance of nintendont should be "Nintendont", not "Nintendon't"

**Comment by playerjmr** [comment]
okay, I believe that should have gone through then

**Pull request merged**